### PR TITLE
New confluence code block parser plugin

### PR DESCRIPTION
**Pull request by Skarlso**
Hi @JohannesKaufmann 

I have the pleasure of working with this library. I had to add a confluence page parser to parse out code blocks. And I thought I'd add it back to you if you like the plugin / you think the changes are appropriate.

Thanks for your great work on this! :)

### CODE REVIEW

File: plugin/confluence_code_block.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import (
+	"fmt"
+	"strings"
+
+	md "github.com/JohannesKaufmann/html-to-markdown"
+	"github.com/PuerkitoBio/goquery"
+)
+
+// ConfluenceCodeBlock converts `<ac:structured-macro>` elements
```
Comment (JohannesKaufmann) [inline plugin/confluence_code_block.go:11]: Can you change the description to something like that?
```suggestion
// ConfluenceCodeBlock converts `<ac:structured-macro>` elements that are used in Atlassian’s Wiki “Confluence”.
// [Contributed by @Skarlso]
```

That makes it a bit clearer that I can't help much when a bug reports for this plugin is submitted😆

**Comment by JohannesKaufmann** [comment]
Thanks a lot for contributing back to the library. Really appreciate that 🙏

The changes look good to me. But I will have a closer look next week (when I have more time), before merging it.

File: plugin/confluence_code_block.go (hunk 1 above)
Comment (Skarlso) [reply plugin/confluence_code_block.go:11]: Oh absolutely, mate. 😊👍

**Comment by Skarlso** [comment]
Done :) Thanks for making it a suggestion! :D I'm lazy it was really cool just pressing the commit button hahaha.

**Comment by JohannesKaufmann** [comment]
@Skarlso thanks again for contributing 🥳

**Comment by Skarlso** [comment]
You're welcome, my man. 😊👍

**Comment by elijahgagne** [comment]
This was something I needed, thanks!

**Comment by Skarlso** [comment]
Oh cool! Excellent. I have one another which is parsing out image tags. I
can add that too. 😊

**Comment by Skarlso** [comment]
Added in https://github.com/JohannesKaufmann/html-to-markdown/pull/18 :)

**Pull request merged**